src/services/ui/src/pages/form/zod.ts

```diff
@@ -36,5 +36,4 @@ export const zAttachmentRequired = ({
 
 export const zAdditionalInfo = z
   .string()
-  .max(4000, "This field may only be up to 4000 characters.")
-  .optional();
+  .max(4000, "This field may only be up to 4000 characters.");
```

Comment:
Leaving `.optional()` up to each schema instead of pre-defining in zod variable.